frontend/src/lib/components/Cards/TextCard/TextCard.tsx

```diff
@@ -61,90 +76,100 @@ export function TextCardInternal(
     const otherDashboards = nameSortedDashboards.filter((dashboard) => dashboard.id !== dashboardId)
     return (
         <div
-            className={clsx('TextCard rounded flex flex-col', className, showResizeHandles && 'border')}
+            className={clsx('Card TextCard rounded flex flex-col', className, showResizeHandles && 'border')}
```

Comment:
Shouldn't we be using the `Card` component? Feels dirty to include a different component's class here.

Reply:
Is it better hidden in the scss? I can see us ending up with a factory that hides `InsightCard`, `TextCard`, and other kinds from the consumer but wanted to not get too derailed here